OneDriveConnector/OneDriveConnector/OneDriveConnector.cs

```diff
@@ -18,8 +18,10 @@ public sealed class OneDriveConnector
         /// <summary>
         /// Is true if currently logged in to OneDrive, false otherwise.
         /// </summary>
-        public bool isLoggedIn { get; private set; } = false;        
-
+        public bool isLoggedIn { get; private set; } = false;
+        public string accessToken { get; private set; } = string.Empty;
```

Comment:
They are lost on app restart. It would be ideal for this class to implement secure storage itself (or at least have the option to), but that isn't strictly needed, so the responsibility is left to the invoking app to manage.